README.md

```diff
@@ -159,6 +159,31 @@ If you want to commit generated artifacts in the release commit (e.g. [#96](http
 "release": "git add <file(s) to commit> && standard-version -a"
 ```
 
+### Use a specific context
+
+If you want to use a specific context, you can use `--context <file.json>` or `-c  <file.json>`
```

Comment:
this seems like a great addition to me; mind adding a note to the README.md with a specific example as to why you'd want to do this, e.g., 

> As an example, if you use a version control system other than GitHub, you might want to change the default URL pattern linking to an issue; see https://github.com/conventional-changelog-archived-repos/conventional-changelog-writer for details.

Reply:
Hello @bcoe,
I've done modifications on the PR can you review it please.
Regards.
Cyril